package.json

```diff
@@ -24,53 +28,70 @@
     "prepack": "yarn build"
   },
   "peerDependencies": {
-    "eslint": "^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0"
+    "eslint": "^3.0.0 || ^4.0.0 || ^5.0.0 || ^6.0.0 || ^7.0.0"
   },
   "devDependencies": {
-    "@types/eslint": "6.1.3",
-    "@types/estree": "0.0.39",
-    "@types/jest": "22.2.2",
-    "@types/lodash": "4.14.106",
+    "@types/eslint": "7.2.0",
+    "@types/estree": "0.0.45",
+    "@types/jest": "26.0.3",
+    "@types/lodash": "4.14.157",
     "@types/minimist": "1.2.0",
-    "@types/node": "12.12.6",
-    "@typescript-eslint/experimental-utils": "2.6.1",
-    "@typescript-eslint/parser": "2.6.1",
-    "babel-eslint": "8.2.2",
-    "eslint": "6.6.0",
-    "eslint-config-prettier": "2.9.0",
-    "eslint-plugin-import": "2.18.2",
-    "eslint-plugin-notice": "0.6.7",
+    "@types/node": "14.0.14",
+    "@typescript-eslint/experimental-utils": "3.5.0",
+    "@typescript-eslint/parser": "3.5.0",
+    "babel-eslint": "10.1.0",
+    "eslint": "7.3.1",
+    "eslint-config-prettier": "6.11.0",
+    "eslint-plugin-import": "2.22.0",
+    "eslint-plugin-notice": "0.9.10",
     "eslint-plugin-sonarjs": "0.5.0",
-    "jest": "24.9.0",
-    "lint-staged": "7.3.0",
-    "lodash": "4.17.5",
-    "minimist": "1.2.0",
-    "prettier": "1.11.1",
-    "rimraf": "2.6.2",
-    "ts-jest": "24.1.0",
-    "ts-node": "5.0.1",
-    "typescript": "3.6.4"
+    "jest": "26.1.0",
+    "lint-staged": "10.2.11",
+    "lodash": "4.17.15",
+    "minimist": "1.2.5",
+    "prettier": "2.0.5",
+    "rimraf": "3.0.2",
+    "ts-jest": "26.1.1",
+    "ts-node": "8.10.2",
+    "typescript": "3.9.5"
   },
   "prettier": {
     "printWidth": 120,
     "trailingComma": "all"
   },
   "jest": {
-    "roots": ["tests", "src"],
-    "collectCoverageFrom": ["src/**/*.ts"],
+    "roots": [
+      "tests",
+      "src"
+    ],
+    "collectCoverageFrom": [
+      "src/**/*.ts"
+    ],
     "globals": {
       "ts-jest": {
         "babelConfig": false
       }
     },
-    "moduleFileExtensions": ["ts", "js", "json"],
+    "moduleFileExtensions": [
+      "ts",
+      "js",
+      "json"
+    ],
     "transform": {
       "^.+\\.ts$": "ts-jest"
     },
-    "testMatch": ["<rootDir>/tests/**/*.test.ts"]
+    "testMatch": [
+      "<rootDir>/tests/**/*.test.ts"
+    ]
   },
   "lint-staged": {
-    "*.{ts,tsx,js}": ["eslint", "prettier --write", "git add"],
-    "*.json": ["prettier --write", "git add"]
-  }
+    "*.{ts,tsx,js}": [
+      "eslint",
+      "prettier --write"
+    ],
+    "*.json": [
+      "prettier --write"
+    ]
+  },
+  "dependencies": {}
```

Comment:
```suggestion
```

Reply:
Thanks for the suggestion. This actually serves two purposes though. It shuts up my package.json linter, avoiding the distraction of its messages. :-)  Of possibly more interest, with `package.json` files allowing these keys to be placed anywhere, it can be helpful to quickly search a file for `dependencies` to find them regardless of location. When one is in fast mode and/or jumping across projects, that can save a small bit of time and effort..

Reply:
Okay, no problem. I thought it was an leftover of a `npm uninstall` which does not get rid of the key either.